src/Main.js

```diff
@@ -57,6 +57,7 @@ const AppContent = () => {
                       name="HomeStack"
                       component={HomeStack}
                       options={{
+                        tabBarStyle: { display: 'none' },
```

Comment:
Perhaps not the best solution, maybe removing createBottomTabNavigator() entirely would be better? Replace it by const Stack = createNativeStackNavigator() as seen in all stacks present in components/stacks/*

Reply:
Ahhhh that's what I was looking for but my react native is kind of rusty 🥲
Done!